### ISSUE TITLE
Get rid of dead nodes

### ISSUE THREAD

**Issue by LuanRT** [LuanRT/YouTube.js#547]
There are a handful of renderers YouTube no longer uses but we still have parsers for them, we should probably start cleaning things up before it gets messy.

**Comment by absidue** [comment]
The old channel external link and like dislike nodes, would be good candidates.

**Comment by LuanRT** [comment]
True. For the new dislike/like models we should prob wait a bit as they are still used sometimes.
